src/app/listSubmissions/listSubmissions.lambda.ts

```diff
@@ -1,12 +1,18 @@
 import { ApiGatewayV2Response } from '@gemeentenijmegen/apigateway-http';
 import { Response } from '@gemeentenijmegen/apigateway-http/lib/V2/Response';
-import { APIGatewayProxyEventV2 } from 'aws-lambda';
+import { APIGatewayProxyEvent } from 'aws-lambda';
 import { ListSubmissionsRequestHandler } from './ListSubmissionsRequestHandler';
-import { parsedEvent } from './parsedEvent';
+import { parsedEvent, parsedEventWithAuthorizer } from './parsedEvent';
 
-export async function handler(event: APIGatewayProxyEventV2): Promise<ApiGatewayV2Response> {
+export async function handler(
+  event: APIGatewayProxyEvent): Promise<ApiGatewayV2Response> {
```

Comment:
Kan dit niet 1 regel zijn? 
Return type is nog v2

Reply:
ik heb de wijzigingen hierin teruggedraaid, vandaar ook die. Maar gezien dat niet klopt wel de moeite even te fixen.

Reply:
behalve dat de responses ook in v2-format zijn. Maak er een los issue van.